src/get_prediction_sex_plot_function.R

```diff
@@ -1,10 +1,10 @@
 library(ggplot2)
 getPredictionSexPlot <- function(TablaDatos, prediccion) {
-  graficaPrediccion <- ggplot(TablaDatos,aes(x=darvic,y=prediccion,color=sexo,shape=sexo)) +
-  geom_point(size=5) +
+  graficaPrediccion <- ggplot(TablaDatos,aes(x = darvic, y = prediccion, color=sexo, shape = sexo)) +
+  geom_point(size = 5) +
```

Comment:
La sugerencia de [aquí](https://style.tidyverse.org/ggplot2.html#long-lines-3) es la siguinete:
```suggestion
      geom_point(size = 5) +
```